scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -104,6 +104,10 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
         config.archive.createFileArchiver()
     }
 
+    private val createMissingArchives by lazy {
+        config.archive?.createMissingArchives ?: false
```

Comment:
Getting this property is very cheap, so there should be no need to evaluate it lazily.